packages/atomic/src/pages/examples/folding.html

```diff
@@ -76,6 +86,365 @@
           <atomic-layout-section section="results">
             <atomic-smart-snippet></atomic-smart-snippet>
             <atomic-folded-result-list image-size="small">
+              <atomic-result-template must-match-sourcetype="Slack">
+                <template>
+                  <style>
+                    div.result-root.with-sections.display-list.density-normal atomic-result-section-visual {
+                      margin-right: 16px;
+                    }
+
+                    .slack-section-badge {
+                      align-items: center;
+                      display: flex;
+                      width: 100%;
+                    }
+
+                    .slack-title-icon {
+                      display: flex;
+                      width: 100%;
+                      margin-top: 8px;
+                    }
+
+                    .slack-badge::part(result-badge-element) {
+                      background-color: #86d8ff;
+                      color: white;
+                      border-radius: 32px;
```

Comment:
If you want the border radius to be half the badge's height, I would recommend to do this:
```css
height: calc(var(--row-height) / 2);
```

I didn't test this, but it should work in theory, since the badge section exposes a `--row-height` variable which is used by the badge.